### DISCUSSION TITLE
docker permissions

### DISCUSSION THREAD

**Discussion by JulianBorghuis** [AzuraCast/AzuraCast#5196] [Q&A]
Question,

I want to have an non root user access to  /var/lib/docker/volumes/azuracast_station_data
 but only read and execute permissions, no write. How can i do this without fucking up my docker permissions.

**Comment by BusterNeece** [comment]
@julianborghuis Really there's no simple way to accomplish this, because our internal SFTP users service doesn't support read-only accounts (currently). It's absolutely possible we could implement that later, though.
